templates/careers/lifestyle.html

```diff
@@ -49,10 +49,7 @@ <h1>Lifestyle</h1>
     <strong>Canonical is unlike any other company in the world. It’s not for everybody, but it is amazing for people who are organized, passionate about technology, and like to travel. People at Canonical say the people at Canonical are amazing. That’s special.</strong>
```

Comment:
There is a suggested change in this sentence too which I agree with. Can you update this one too please.